engine/existing_test.go

```diff
@@ -136,3 +136,339 @@ func TestRangeQuery(t *testing.T) {
 		})
 	}
 }
+
+func TestMinMaxOverTimeRangeQuery(t *testing.T) {
```

Comment:
Let's remove these additional tests, and keep only the ones in `engine_test`. It should be sufficient to have comparisons with the old engine only.